manifests/role/umich_mailserver.pp

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# The minimum configuration for a host that logically belongs to the
+# University of Michigan. Most--but not all--other roles rely on this
+# one.
```

Comment:
Copypasta error

Reply:
Ha! I didn't even look at the comments, good catch

Reply:
I can edit that so its not worded the same the umich role.